dramatiq/actor.py

```diff
@@ -220,3 +152,74 @@ def __repr__(self):
 
     def __str__(self):
         return "Actor(%(actor_name)s)" % vars(self)
+
+
+def actor(fn=None, *, actor_name=None, queue_name="default", priority=0, broker=None, actor_class=Actor, **options):
```

Comment:
Tiny nitpick: please move `actor_class` before `actor_name`. All of those are keyword-only arguments so it's not a breaking change and I like how it reads a little better.

Reply:
Done.